src/schema/sources.ts

```diff
@@ -557,11 +659,18 @@ export const resolvers: IResolvers<any, Context> = {
         (node, index) =>
           membershipsPageGenerator.nodeToCursor(page, args, node, index),
         (builder) => {
+          const roleRankQuery = `
```

Comment:
I suggest using the same way to calculate the role rank everywhere. Sometimes you use SQL and sometimes you use the array. Please stick to the latter.